### DISCUSSION TITLE
image cdn discussion / feedback / issues

### DISCUSSION THREAD

**Discussion by cameronbraid** [gatsbyjs/gatsby#35126] [Ideas / Feature Requests]
I am experimenting with gatsby image-cdn and have the following feedback

* I really like it so far -  makes image heavy sites build soo much faster... great work. :) 
* base64 is not suitable for the `url` and `params` parameters for `_gatsby/image/:url/:params` as base64 encoded strings can contain `/`.  base64url is a suitable alternative https://github.com/nodejs/node/issues/26512
* please add support for transformOptions.trim in the  new `gatsbyImage` resolver

**Comment by wardpeet** [comment]
Thank you! I wasn't aware of base64url, but that's precisely what we need. We will publish a new version with fixes in the coming days.
